content/en/docs/tasks/job/automated-tasks-with-cron-jobs.md

```diff
@@ -93,7 +93,7 @@ Note that the job name and pod name are different.
 
 ```shell
 # Replace "hello-4111706356" with the job name in your system
-$ pods=$(kubectl get pods --show-all --selector=job-name=hello-4111706356 --output=jsonpath={.items..metadata.name})
+$ pods=$(kubectl get pods --all-namespaces --selector=job-name=hello-4111706356 --output=jsonpath={.items..metadata.name})
```

Comment:
Actually, just removing `--show-all` would be OK here. You don't need to use `--all-namespaces` instead in this context.
/cc @deads2k

Reply:
I agree with @xiangpengzhao Reading the deprecation notes the proper correction is to just remove the --show-all from the kubectl command.